packages/edit-site/src/components/post-fields/index.js

```diff
@@ -344,6 +344,16 @@ function usePostFields( viewType ) {
 					return <time>{ getFormattedDate( item.date ) }</time>;
 				},
 			},
+			{
+				id: 'comment_status',
+				label: __( 'Discussion' ),
+				type: 'text',
+				editAs: 'radio',
```

Comment:
The REST API doesn't define any boolean data type, so we'll have to wait to implement that.

I took this as an opportunity to explore how to provide a different edit component (radio) than the default (select), hence the `editAs` prop. I also considered the following:

1. We only provide `select` as default and consumer can provide its own `edit` function. This seems too basic.
2. Alternative names for the prop: `edit`, overloading the custom `edit` function the consumer can provide — could be nice.

Reply:
>The REST API doesn't define any boolean data type, so we'll have to wait to implement that.

What do you mean with this?

Reply:
That all data in https://developer.wordpress.org/rest-api/reference/pages/#schema is string, integer, or object.

Reply:
Yeah, for pages it doesn't. But what will we have to wait to implement?